pkgs/tools/archivers/lha/default.nix

```diff
@@ -0,0 +1,28 @@
+{stdenv, lib, fetchFromGitHub, autoreconfHook}:
+
+stdenv.mkDerivation {
+  name = "lha-1.14i";
+
+  src = fetchFromGitHub {
+    owner = "jca02266";
+    repo = "lha";
+    rev = "03475355bc6311f7f816ea9a88fb34a0029d975b";
```

Comment:
I don't think the program is versioned very well. We should use the unstable-XXXX-XX-XX format instead.